ci_framework/roles/copy_container/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 
 - name: Build the copy-container
   register: go_build
-  ansible.builtin.command:
+  ci_script:
     cmd: go build
     chdir: "{{ temporary_copy_container_dir.path }}"
```

Comment:
you're missing the "output_dir" parameter, needed to nudge reproducer and log files.

The output_dir value is usually `{{ ROLE_basedir }}/artifacts`, as you can see in this already implemented call:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/artifacts/tasks/crc.yml#L44-L53

Reply:
thank you Cedric